manageiq-operator/pkg/helpers/miq-components/httpd_conf.go

```diff
@@ -432,3 +433,57 @@ LimitRequestFieldSize 524288
 `
 	return fmt.Sprintf(s, protocol)
 }
+
+func apiHttpdConfig(protocol string) string {
+	s := `
+## ManageIQ HTTP Virtual Host Context
+
+Listen 3000
+
+# Timeout: The number of seconds before receives and sends time out.
+Timeout 120
+ServerSignature Off
+ServerTokens Prod
+
+RewriteEngine On
+Options SymLinksIfOwnerMatch
+
+# LimitRequestFieldSize: Expand this to a large number to allow pass-through.
+#   This does not introduce a potential DoS, because the value is validated by
+#   the httpd container first.  However, if a user changes this value in the
+#   httpd container, we need to be able to accomodate that value here also.
+LimitRequestFieldSize 524288
+
+<VirtualHost *:3000>
+  IncludeOptional conf.d/*_config
+
+  ServerName %s://web-service
```

Comment:
Oh this line is different too.  Wonder if it's better to have a method that takes the server name and builds this string, if this is the only change.